doc/sphinx/Pacemaker_Explained/operations.rst

```diff
@@ -38,113 +38,160 @@ two operations for the same resource with the same name and interval.
-   |                |                                   | (see :ref:`s-operation-defaults`).  Allowed values: |
-   |                |                                   | ``true``, ``false``.                                |
-   +----------------+-----------------------------------+-----------------------------------------------------+
-   | role           |                                   | .. index::                                          |
-   |                |                                   |    single: role; action property                    |
-   |                |                                   |    single: action; property, role                   |
-   |                |                                   |                                                     |
-   |                |                                   | Run the operation only on node(s) that the cluster  |
-   |                |                                   | thinks should be in the specified role. This only   |
-   |                |                                   | makes sense for recurring ``monitor`` operations.   |
-   |                |                                   | Allowed (case-sensitive) values: ``Stopped``,       |
-   |                |                                   | ``Started``, and in the case of :ref:`promotable    |
-   |                |                                   | clone resources <s-resource-promotable>`,           |
-   |                |                                   | ``Unpromoted`` and ``Promoted``.                    |
-   +----------------+-----------------------------------+-----------------------------------------------------+
+   :widths: 2 2 3 4
+   :header-rows: 1
+
+   * - Name
+     - Type
+     - Default
+     - Description
+   * - .. _op_id:
+       
+       .. index::
+          pair: op; id
+          single: id; action property
+          single: action; property, id
+       
+       id
+     - :ref:`id <id>`
+     - 
+     - A unique identifier for the XML element *(required)*
+   * - .. _op_name:
+       
+       .. index::
+          pair: op; name
+          single: name; action property
+          single: action; property, name
+       
+       name
+     - :ref:`text <text>`
+     - 
+     - An action name supported by the resource agent *(required)*
+   * - .. _op_interval:
+       
+       .. index::
+          pair: op; interval
+          single: interval; action property
+          single: action; property, interval
+       
+       interval
+     - :ref:`duration <duration>`
+     - 0
+     - A positive value directs Pacemaker to schedule instances of the action
+       that recur at the given interval, which makes sense only with ``name``
+       set to :ref:`monitor <s-resource-monitoring>`. A value of 0 directs
+       Pacemaker to apply any other properties configured for this action to
+       nonrecurring instances scheduled as needed. *(required)*
```

Comment:
Still a bit dense... not sure if it can be improved further without losing accuracy. Maybe something like this? (Main change is dropping "instances")

"A positive value directs Pacemaker to schedule recurrences of the action at the given interval, which makes sense only with ``name`` set to :ref:`monitor <s-resource-monitoring>`. A value of 0 directs Pacemaker to schedule the action as needed (not recurring at regular intervals), still applying any other properties configured for this action."

Reply:
I think I can get it a little better

Reply:
By the way, have we ever thought about making `interval` optional? Would make at least as much sense as `0` for non-monitor actions.

Reply:
Actually I think it does default to 0 already

Reply:
Probably, but the schema currently requires that it be set explicitly.